test/default.nix

```diff
@@ -51,7 +51,9 @@ let
       "bootstrap"
       "new"
       "new-static"
-      "new-static-github"
+      "init"
+      "init-static"
+      "init-static-github"
```

Comment:
It's enough to have `static*` tests only for one command variant – they only test that the build and download of the executable works.